### PR TITLE
Replaced lodash.flattendeep with lodash

### PR DESCRIPTION
**Pull request by chhavin**
PR for issue : Min lodash version should be updated to 4.17.5 #16

### CODE REVIEW

**Comment by chhavin** [comment]
@aweary Can you please review. Thanks.

**Review by ljharb — CHANGES_REQUESTED** [review]
If were replacing this, let’s replace it with https://npmjs.com/array.prototype.flat instead

**Comment by aweary** [comment]
Yeah I don't want to pull in all of lodash just for this

File: src/grammar.ne [hunk 1]
```diff
@@ -1,8 +1,11 @@
 @{%
-  const flattenDeep = require('lodash/flattendeep');
   const appendItem = (a, b) => d => d[a].concat([d[b]]);
   const appendItemChar = (a, b) => d => d[a].concat(d[b]);
 
+  const flattenDeep = arr => {
```
Comment (ljharb) [inline src/grammar.ne]: why not use https://npmjs.com/array.prototype.flat and `flat(arr, Infinity)`, since it's a standard feature in the language now?

Comment (chhavin) [reply src/grammar.ne]: I guess I just preferred using the native javascript but anyways replaced with array.prototype.flat now.

**Review by ljharb — APPROVED** [review]

File: src/grammar.ne (hunk 1 above)
Comment (ljharb) [reply src/grammar.ne]: That package will use native JS in a newer engine, and will have a fallback impl in an older one. Thanks!

**Comment by chhavin** [comment]
Can someone with write access please merge the changes @ljharb @aweary . Thanks.

**Pull request merged**